src/sentry/integrations/github/urls.py

```diff
@@ -10,4 +12,9 @@
         GitHubSearchEndpoint.as_view(),
         name="sentry-extensions-github-search",
     ),
+    url(
+        r"^configure/(?P<signed_params>[^\/]+)/$",
```

Comment:
@AniketDas-Tekky shouldn't this just be a query param: `?signed_params=< signed_params>`

Not in the URL itself, see what we do for Vercel for example.